tests/unit/Infrastructure/Manage/Dto/ManageEntityTest.php

```diff
@@ -102,4 +103,29 @@ public function test_diff_oidc()
         // 'provide everything' manner.
         $this->assertCount(2, $diffResults['metaDataFields.redirectUrls']);
     }
+
+    public function test_is_not_status_publication_requested()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
+        $this->assertFalse($entity->isStatusPublicationRequested());
+    }
+
+    public function test_is_status_publication_requested()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
+        $entity->updateStatus(Constants::STATE_PUBLICATION_REQUESTED);
+        $this->assertTrue($entity->isStatusPublicationRequested());
+    }
+
+    public function test_is_requested_production_entity_copy()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
+        $this->assertTrue($entity->isRequestedProductionEntity(true));
+    }
+
+    public function test_is_requested_production_entity_no_copy()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
+        $this->assertFalse($entity->isRequestedProductionEntity(false));
+    }
```

Comment:
Consider merging these tests into the test_is_not_status_publication_requested that way you can save up on a couple of clock cycles in building the manage entities from json. You might have to rename the test name though.